articles/deconstructing-shared-nothing-and-shared-disk-clustering-architectures/index.md

```diff
@@ -0,0 +1,66 @@
+
+It is perfect for sectors in an industry that use data that cannot be partitioned, and it needs to be in one place for processing. And these syndicates should require only a limited transfer of data at a time.
+
+On the downside, the shared-disk clustering architecture model is not as scalable as shared-nothing clustering since each node updates at a time. The way to get around this downside is by using optimization techniques that enable large-scale businesses to expand without a problem.
+
+### Differences between Shared Nothing and Shared Disk Clustering Architectures
+
+#### Scalability according to performance
+
+Shared-nothing is very practical to expand according to the size of a business. And that is because you can easily add a new node into the already existing cluster. The disadvantage of scaling up is that performance may decrease significantly because the nodes have to cross-communicate and coordinate data back and forth. This means you can only expand to a limited level before you see a drop in performance.
+
+Shared-disk, as mentioned above, is challenging when it comes to scaling up. And that is because the data transmitted need not be partitioned. Even so, when the right optimization steps are taken, the process of data transmission steadily rises, and the scalability opportunity is unlimited.
+
+#### Scalability according to capacity
+
+Shared-nothing is easy to scale up in terms of capacity. You have to add a new node to the cluster to get more space.
+
+The shared disk clustering architecture, which uses one disc device with separate memories, can be upgraded memory-wise, increasing the capacity. Data transmission speed is not affected in any way.
+
+#### Availability
+
+Shared-nothing clustering architecture is highly available. This is attributed to the fact that it can scale up quickly and cheaply. The availability of shared-disk can be limiting because the hardware used is costly compared to the shared-nothing clustering architecture.
+
+#### Data storage
+
+Shared-nothing clustering deals with data that is independently stored in separate nodes, which means that the data being stored can be partitioned and summoned when needed. Shared-disk works with information that cannot be separated.
+
+#### Load balancing
+
+[Load balancing](https://en.wikipedia.org/wiki/Load_balancing_(computing)#) is the process that a computer or processor uses to distribute tasks over specific system resources. The aim is usually to make this transmission more efficient. On that note, shared-nothing clustering architecture has a fixed load balance, whereas the shared-disk clustering architect has a dynamic loading balance, which is progressive.
+
```

Comment:
Add a table summarizing the features of the 2 types of clustering. That makes this article crisp and clear.